x-pack/legacy/plugins/fleet/common/types/domain_data.ts

```diff
@@ -4,10 +4,126 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import * as t from 'io-ts';
+import { AGENT_TYPE_EPHEMERAL, AGENT_TYPE_PERMANENT, AGENT_TYPE_TEMPORARY } from '../constants';
 export { Policy, Datasource, Status, Output } from '../../../ingest/server/libs/types';
```

Comment:
Interesting that it doesn't complain about this. If it passes, I'm ok with leaving it for later, but I think we'll eventually need to move the shared Ingest types to `ingest/common/types` or something similar so the client & server can both use them.

Reply:
Yes it's weird that eslint is not complaining, I will keep it like this for later